src/frontend/src/components/ProfileForm.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
+  export let private_key;
+  export let pk;
+  export let username;
+  export let about;
+  export let avatar_url;
+  export let loading;
+  export let submitFunction;
```

Comment:
submit_function, nech to máme jednotné v našom kóde. Alebo všetko prepísať do camel casu :D